packages/flutter/lib/src/material/ink_highlight.dart

```diff
@@ -83,23 +83,23 @@ class InkHighlight extends InteractiveInkFeature {
   late AnimationController _alphaController;
 
   /// Whether this part of the material is being visually emphasized.
-  bool get active => _active;
-  bool _active = true;
+  bool get showing => _showing;
+  bool _showing = true;
 
   /// Start visually emphasizing this part of the material.
-  void activate() {
-    _active = true;
+  void show() {
+    _showing = true;
     _alphaController.forward();
   }
 
   /// Stop visually emphasizing this part of the material.
-  void deactivate() {
-    _active = false;
+  void hide() {
```

Comment:
same question here... seems like this will almost certainly affect someone, so if there's not a strong reason to do it...